### PR TITLE
PROGRAMMING LANGUAGES- PYTHON-ADDED PYTHON PROJECTS SECTION

### PR DESCRIPTION
**Pull request by Varsha010101**
### Description: 
It looks forward to cover projects in beginner, intermediate and advance level a user can show to build up his resume. It will cover project topics like:
Calculator
chatbot
machine learning model
banking system
and many more

- https://realpython.com/build-a-chatbot-python-chatterbot/
- https://www.geeksforgeeks.org/real-time-translation-app-using-python/
- https://github.com/topics/python-blockchain

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Varsha010101** [comment]
Kindly review this PR and label it under GSSoC @jfmartinz. Refers to issue number #401

File: Programming Languages/Java/readme.md [hunk 1]
```diff
@@ -946,7 +946,11 @@ Here's the updated table:
   </tr>
   <tr>
     <td><a href="https://medium.com/automation-generation/build-an-automated-stock-trading-system-in-java-bfb5b739eb6f">Cointrader</a></td>
-    <td>Coin Trader is a Java-based backend for algorithmically trading cryptocurrencies. It provides data collection and export, complex event processing and triggering, and backtesting - paper trading - live trading..</td>
+
+  <td>Coin Trader is a Java-based backend for algorithmically trading cryptocurrencies. It provides data collection and export, complex event processing and triggering, and backtesting - paper trading - live trading..</td>
```
Comment (jfmartinz) [inline Programming Languages/Java/readme.md:950]:
```suggestion

```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Another change is that you removed the Youtube Playlist and Conclusion:

image

Add your pyton projects before conclusion

**Comment by Varsha010101** [comment]
@jfmartinz All mentioned changes have been updated. Please review this PR and merge it.

**Comment by Varsha010101** [comment]
@jfmartinz All mentioned changes have been updated. Please review this PR and merge it.

**Pull request closed without merging**